wavefront/server/modules/knowledge_base_module/knowledge_base_module/controllers/rag_retreival_controller.py

```
@@ -228,6 +228,30 @@ async def retrieve_query(
+            'document_date_start, document_date_end, and exclude_filter4. '
+            'The underlying query never touches the HNSW index (see '
+            'QueryGenerator.get_image_embedding_dino_exact_match) so results '
+            'are exact, not approximate.'
+        ),
+    ),
+    filter1: Optional[str] = Query(
+        None, description='Exact-match only: equality filter on knowledge_base_documents.filter1'
+    ),
+    document_date_start: Optional[datetime] = Query(
+        None, description='Exact-match only: start of the document_date window (inclusive)'
+    ),
```

Ruff (0.16.3)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)